dnf/module/repo_module_version.py

```diff
@@ -148,12 +148,7 @@ def artifacts(self):
         return self.module_metadata.get_rpm_artifacts().get()
 
     def requires(self):
-        dependencies = self.module_metadata.get_dependencies()
-
-        if dependencies:
-            return dependencies[0].get_requires()
-
-        return {}
+        return self.module_metadata.peek_requires()
```

Comment:
note that you need to bump min version of libmodulemd

Reply:
good point, thanks